node-rti/src/lib.rs

```diff
@@ -48,6 +48,7 @@ pub struct ExecutionReport {
     pub gas_refunds: Vec<(H256, u64)>,
     pub gas_charges: Vec<(H256, u64)>,
     pub gas_transfers: Vec<(H256, H256, u64)>,
+    pub outcomes: Vec<(H256, Result<(), Vec<u8>>)>,
```

Comment:
```suggestion
    pub outcomes: Vec<(H256, Result<(), String>)>,
```

why not just string?